classes/author.rb

```diff
@@ -1,9 +1,9 @@
 require_relative './item'
-class Author
+class Author < Item
```

Comment:
according to the UML diagram given, the Author class share a relationship with the Item class and it not a sub-class of Item class, please check it out and please make a change by removing the inheritance on class Author 
<img width="755" alt="review" src="https://user-images.githubusercontent.com/108295038/234359355-563173b4-f2ec-4060-8d11-5c3270ca34b8.png">